+++
# Experience widget.
widget = "experience"  # See https://sourcethemes.com/academic/docs/page-builder/
headless = true  # This file represents a page section.
active = true  # Activate this widget? true/false
weight = 50 # Order that this section will appear.

title = "Journey so far"
subtitle = ""

# Date format for experience
#   Refer to https://sourcethemes.com/academic/docs/customization/#date-format
date_format = "Jan 2006"

# Experiences.
#   Add/remove as many `[[experience]]` blocks below as you like.
#   Required fields are `title`, `company`, and `date_start`.
#   Leave `date_end` empty if it's your current employer.
#   Begin/end multi-line descriptions with 3 quotes `"""`.

[[experience]]
  title = "Principal Investigator"
  company = "Incyte"
  company_url = "https://www.incyte.com"
  location = "Wilmington, DE"
  date_start = "2022-02-28"
  date_end = ""
  description = """
  Responsibilities include:
    Check back later!
    """




[[experience]]
  title = "Research Biostatistician II"
  company = "Memorial Sloan Kettering Cancer Center"
  company_url = "https://www.mskcc.org/departments/epidemiology-biostatistics"
  location = "New York"
  date_start = "2018-01-31"
  date_end = ""
  description = """
  Responsibilities include:
  
  *  Developed [survClust](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-020-00804-8), a semi-supervised classification algorithm that maximizes survival differences in molecularly profiled cohorts on various platforms. This is implemented through a weighted clustering model, utilizing multidimensional scaling. Classification groups are deemed robust via cross-validation.
  *  survClust was then used in a pancancer cohort of patients treated with immune checkpoint blockade therapies to stratify patients with worst prognisis. Read more [here](https://www.science.org/doi/10.1126/scitranslmed.abf5107?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed)
  * Lead genomics analyst of the International consortium of Melanoma ([InterMEL](https://intermel.org)) and working on various predictive models based on multi-omic data and identifying germline calls from tumor-only somatic mutation calling pipeline
"""

[[experience]]
  title = "Research Biostatistician"
  company = "Memorial Sloan Kettering Cancer Center"
  company_url = ""
  location = "New York"
  date_start = "2014-12-01"
  date_end = "2018-01-31"
  description = """
  Responsibilities include:
  *  Integrated analysis of various cancer types as part of The Cancer Genome Atlas (TCGA) consortium like Liver Hepatocellular Carcinoma (LIHC), Prostate Adenocarcinoma (PRAD) and Skin Cutaneous Melanoma (SKCM) using joint latent variable model implemented in iCluster, to arrive at molecularly distinct subtypes.
  * Providing genomics and analytical support to faculty members of Epidemiology and Biostatistics Department at Memorial Sloan Kettering Cancer Center on a broad range of analysis like copy number and clonal evolution, mutational signature analysis, and building statistical models to identify prognostic molecular features in exome sequencing and mutation panel testing datasets.
"""
[[experience]]
  title = "Assistant Research Biostatistician"
  company = "Memorial Sloan Kettering Cancer Center"
  company_url = ""
  location = "New York"
  date_start = "2012-05-01"
  date_end = "2014-12-31"
  description = """
  Responsibilities include:
  *  Developed a somatic mutation caller by applying Random Effects model. Tested on publically available and in-house datasets.
  *  Understanding etiological tumor heterogeneity across various molecular assays like gene expression, mutation, copy number, and epigenetic data through known clinical risk factors to characterize distinct risk groups.
  *  Developed a validated prognostic gene risk score of colorectal cancer liver metastasis patients.
"""

[[experience]]
  title = "Graduate Research Assistant"
  company = "University of Pittsburgh"
  company_url = ""
  location = "Pittsburgh"
  date_start = "2009-10-01"
  date_end = "2012-04-01"
  description =  """
  
  **Benos Lab**
  * Development of miRNA-seq Illumina pipeline in Perl, involving quality filtering of reads, trimming of adapters, and aligning to the Genome to obtain raw reads for analysis and for the analysis of potential miRNA targetable genes. Understanding miRNA targets and the effect of co-operativity of miRNA via thermodynamics on CLIPSeq data.
  
  **Faeder Lab**
  * Coded a rule-based programming language translator between two rule based modeling languages BioNetGen (BNG) to Stochastic Simulation Compiler (SSC).  
  * Effectively managed code testing, resolving bugs nd releasing distributions of BioNetGen (BNG).
  *  Understanding and modeling of Toll Like Receptor 4 (TLR4) pathway via rule-based models to mimic preconditioning by repeated simulations and identify teh origins of memory effect in TLR4 response to bacterial infection. 
  """

  
+++
