+++
# Experience widget.
widget = "experience"  # See https://sourcethemes.com/academic/docs/page-builder/
headless = true  # This file represents a page section.
active = true  # Activate this widget? true/false
weight = 50 # Order that this section will appear.

title = "Journey so far"
subtitle = ""

# Date format for experience
#   Refer to https://sourcethemes.com/academic/docs/customization/#date-format
date_format = "Jan 2006"

# Experiences.
#   Add/remove as many `[[experience]]` blocks below as you like.
#   Required fields are `title`, `company`, and `date_start`.
#   Leave `date_end` empty if it's your current employer.
#   Begin/end multi-line descriptions with 3 quotes `"""`.

[[experience]]
  title = "Principal Investigator"
  company = "Incyte"
  company_url = "https://www.incyte.com"
  location = "Wilmington, DE"
  date_start = "2022-02-28"
  date_end = ""
  description = """
  Responsibilities include:
    My role at Incyte is highly cross-functional where I work with scientists from Discovery, Pharmacology and Biology to power their biological hypothesis with data. This exposed me to various view points sometimes of the same problem and in identifying answers supported by data to key questions in drug discovery and translation. I also lead Target Identification and Validation efforts with the help of Machine Learning and other deep learning models.
    """

[[experience]]
  title = "Research Biostatistician"
  company = "Memorial Sloan Kettering Cancer Center"
  company_url = "https://www.mskcc.org/departments/epidemiology-biostatistics"
  location = "New York"
  date_start = "2012-05-01"
  date_end = "2022-02-18"
  description = """
  Responsibilities include:
  
  *  Developed [survClust](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-020-00804-8), a semi-supervised classification algorithm that stratifies patients into cohorts driven by their genetic background and survival. 
  *  survClust was then used in a pancancer cohort of patients treated with immune checkpoint blockade therapies to stratify patients with worst prognosis. Read more [here](https://www.science.org/doi/10.1126/scitranslmed.abf5107?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed)
  * Lead genomics analyst of the International consortium of Melanoma ([InterMEL](https://intermel.org)) and building a framework for identifying false positives from tumor-only somatic mutation calling pipeline. ([Glitter](https://github.com/arorarshi/Glitter))
  *  Integrated analysis of various cancer types as part of The Cancer Genome Atlas (TCGA) consortium like Liver Hepatocellular Carcinoma (LIHC), Prostate Adenocarcinoma (PRAD) and Skin Cutaneous Melanoma (SKCM) using joint latent variable model implemented in iCluster, to arrive at molecularly distinct subtypes.
  * Providing genomics and analytical support to faculty members of Epidemiology and Biostatistics Department at Memorial Sloan Kettering Cancer Center on a broad range of analysis like copy number and clonal evolution, mutational signature analysis, and building statistical models to identify prognostic molecular features in exome sequencing and mutation panel testing datasets.
  *  Understanding etiological tumor heterogeneity across various molecular assays like gene expression, mutation, copy number, and epigenetic data through known clinical risk factors to characterize distinct risk groups.
  *  Developed a validated prognostic gene risk score of colorectal cancer liver metastasis patients.
"""
+++
